R/helper_exec.R

```diff
@@ -23,6 +23,9 @@ future_map = function(n, FUN, ..., MoreArgs = list()) {
   if (getOption("mlr3.debug", FALSE)) {
     lg$info("Running experiments sequentially in debug mode with %i iterations", n)
     mapply(FUN, ..., MoreArgs = MoreArgs, SIMPLIFY = FALSE, USE.NAMES = FALSE)
+  } else if (requireNamespace("mirai", quietly = TRUE) && mirai::status()$connections) {
```

Comment:
This only works with the default compute profile

Reply:
Yes, I didn't want to make it too complicated. What would be the best way to pass the compute profile? We also have to pass it to `mirai_map()` .

Reply:
I suggest we expose a `.compute` argument to `benchmark()` and `resample()`.

Reply:
I'm not so happy with that because we are introducing parameters for a special backend. Moreover, this argument would then have to be added to all tuning and feature selection functions.

Reply:
I wouldn't worry about this. Already in a `with(daemons(...), { })` scope `mirai()` calls will default to those settings with `.compute = NULL`. More to come on this front.